src/main/java/com/studypals/domain/memberManage/api/AuthController.java

```diff
@@ -66,11 +66,13 @@ public ResponseEntity<Response<JwtToken>> signIn(@Valid @RequestBody SignInReq r
 
     @PostMapping("/refresh")
     public ResponseEntity<Response<ReissueTokenRes>> refreshToken(
-            @Valid @RequestBody TokenReissueReq req, @RequestHeader("Authorization") String accessToken) {
-        JwtToken token = JwtToken.builder()
-                .accessToken(accessToken)
-                .refreshToken(req.refreshToken())
-                .build();
+            @Valid @RequestBody TokenReissueReq req,
+            @RequestHeader("Authorization") String accessToken) {
+        JwtToken token =
+                JwtToken.builder()
+                        .accessToken(accessToken.substring(7))
```

Comment:
[P3]

현재 PR에서 남기긴 좀 안 맞긴 한데, 지금 보다 보니 갑자기 떠올라서 코멘트 남겨봅니다,, 뒷북이라 P3으로 하겠습니다.

그러고보니 코드에서 매직 넘버(하드 코딩된 숫자나 문자열)는 가능한 지양하는 것이 좋다는 말을 많이 들었습니다.
해당 라인에선는 7이 매직 넘버가 될텐데, 이게 "Bearer "의 길이라는 건 암묵적으로 알지만
코드 상에서 명확하게 표현하려면 별도의 상수로 표현하는 것이 좋을 것 같습니다.

그런데 이게 변경하기 전 원래 있었던 service 단이나, 현재 controller 단에 있는 것도 애매한 것 같아서,
`JwtToken` 내부에 상수로 두고 사용하는 건 어떨까요?

```java
class JwtToken {
    public static final String BEARER_PREFIX = "Bearer ";
    public static final int BEARER_PREFIX_LENGTH = BEARER_PREFIX.length();
}
```

그리고 사용할 땐 이렇게 사용하는 것 어떠신가요?
```suggestion
.accessToken(accessToken.substring(JwtToken.BEARER_PREFIX_LENGTH));
```

그래서 지금 PR에선 위치를 잡을 수 없는데, `JwtAuthenticationFilter`에도 해당 부분이 있더라구요.
거기서도 저렇게 상수를 사용하면 어떨까 싶습니다.